Resources/Prototypes/Entities/Mobs/Species/arachnid.yml

```
@@ -127,6 +127,7 @@
   - type: FootPrints # Backmen-footsteps
     leftBarePrint: "footprint-left-bare-spider"
     rightBarePrint: "footprint-right-bare-spider"
+  - type: MutilatedCorpse #DS14 - MutilatedCorpse
```

```suggestion
  - type: MutilatedCorpse # DS14 - MutilatedCorpse
```
И пробелы после `#` везде